README.md

```diff
@@ -1,6 +1,6 @@
 | Sonar Code Quality | DockerHub Release | DockerHub Release Candidate | DockerHub Develop |
 |-----|-----|-----|-----|
- [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=usdot-fhwa-stol_carma-web-ui&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=usdot-fhwa-stol_carma-web-ui) | [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/usdotfhwastol/carma-web-ui?label=Carma-Web-UI)](https://hub.docker.com/repository/docker/usdotfhwastol/carma-web-ui) | [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/usdotfhwastolcandidate/carma-web-ui?label=Carma-Web-UI)](https://hub.docker.com/repository/docker/usdotfhwastolcandidate/carma-web-ui) | [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/usdotfhwastoldev/carma-web-ui?label=Carma-Web-UI)](https://hub.docker.com/repository/docker/usdotfhwastoldev/carma-web-ui)
+ [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=usdot-fhwa-stol_carma-web-ui&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=usdot-fhwa-stol_carma-web-ui) | [![Docker Hub build](https://github.com/usdot-fhwa-stol/carma-web-ui/actions/workflows/dockerhub.yml/badge.svg?branch=carma-system-4.5.0)](https://github.com/usdot-fhwa-stol/carma-web-ui/actions/workflows/dockerhub.yml) | [![Docker Hub build](https://github.com/usdot-fhwa-stol/carma-web-ui/actions/workflows/dockerhub.yml/badge.svg?branch=release%2Flavida)](https://github.com/usdot-fhwa-stol/carma-web-ui/actions/workflows/dockerhub.yml) | [![Docker Hub build](https://github.com/usdot-fhwa-stol/carma-web-ui/actions/workflows/dockerhub.yml/badge.svg?branch=develop)](https://github.com/usdot-fhwa-stol/carma-web-ui/actions/workflows/dockerhub.yml)
```

Comment:
Looks like we decided on removing the candidate and release badges and keep develop at least. I will leave it up to you if you want to keep master as well. master is essentially "carma-system" tagges ones anyways, so we can also omi in my opinion